packages/host/app/components/operator-mode/code-submode/module-inspector.gts

```diff
@@ -490,6 +498,10 @@ export default class ModuleInspector extends Component<ModuleInspectorSignature>
       />
     {{/if}}
 
+    <ReportUserContextForAiAssistant
```

Comment:
Properties like `selectedAccordionItem` are local component state, so there is a question of how to read this where we send this context to matrix. We could put it into a service but that would include refactoring in several different places. I opted into created a `ReportUserContextForAiAssistant` component which we can stick into any template where we are interested in gathering pieces of data that is sent to AI.

Reply:
More info here: https://github.com/cardstack/boxel/pull/2622/files#diff-74107da2f57a38bed49ccc37a3cac7b46723735b154806573023d5262a6fcabcR33